taxi/urls.py

```diff
@@ -1,14 +1,100 @@
 from django.urls import path
 
-from .views import index, CarListView, CarDetailView, DriverListView, DriverDetailView, ManufacturerListView
+from .views import (
+    index,
+    CarListView,
+    CarCreateView,
+    CarDetailView,
+    CarUpdateView,
+    CarDeleteView,
+    DriverListView,
+    DriverDetailView,
+    DriverCreateView,
+    DriverDeleteView,
+    DriverLicenceUpdateView,
+    ManufacturerListView,
+    ManufacturerCreateView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
+    driver_to_car,
+)
 
 urlpatterns = [
     path("", index, name="index"),
-    path("manufacturers/", ManufacturerListView.as_view(), name="manufacturer-list"),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
-    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail")
+    path(
+        "manufacturers/",
+        ManufacturerListView.as_view(),
+        name="manufacturer-list"
+    ),
+    path(
+        "manufacturer/create/",
+        ManufacturerCreateView.as_view(),
+        name="create-manufacturer"
+    ),
+    path(
+        "manufacturer/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturer/<int:pk>/delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
+    path(
+        "cars/",
+        CarListView.as_view(),
+        name="car-list"),
+    path(
+        "cars/<int:pk>/",
+        CarDetailView.as_view(),
+        name="car-detail"
+    ),
+    path(
+        "cars/create/",
+        CarCreateView.as_view(),
+        name="create-car"
```

Comment:
It's really not a big deal, but if you put car at the first place everywhere in names, it's better to put car at the first here too